cmd/influxd/run.go

```diff
@@ -306,6 +306,14 @@ func (cmd *RunCommand) CheckConfig() {
 func (cmd *RunCommand) Open(config *Config, join string) *Node {
 	if config != nil {
 		cmd.config = config
+
+		// Override config properties.
+		if cmd.hostname != "" {
+			cmd.config.Hostname = cmd.hostname
+		}
+		cmd.node.hostname = cmd.config.Hostname
+	} else {
+		panic("repeated calls to RunCommand.Open()")
```

Comment:
I'm not sure I understand the purpose of this line at all.  Both why we would panic or the actual message itself.